mep/common/views.py

```diff
@@ -43,6 +47,56 @@ def dispatch(self, request, *args, **kwargs):
         return response
 
 
+class RdfViewMixin(ContextMixin):
+    '''View mixin to add an RDF linked data graph to context for use in serializing
+    and embedding structured data in templates.'''
+
+    #: default schema.org type for a View
+    rdf_type = SCHEMA_ORG.WebPage
+    #: breadcrumbs, used to render breadcrumb navigation. they should be a list
+    #: of tuples like ('Title', '/url')
+    breadcrumbs = []
+    #: json-ld context for the rdf graph; defaults to schema.org
+    json_ld_context = str(SCHEMA_ORG)
+
+    def get_context_data(self, *args, **kwargs):
+        '''Add generated breadcrumbs and an RDF graph to the view context.'''
+        context = super().get_context_data(*args, **kwargs)
+        context['rdf'] = self.as_rdf().serialize(format='json-ld', auto_compact=True,
```

Comment:
Do you know, is this hitting `schema.org` every time we serialize the json-ld? It seems like it would be nice to cache the context if we can, although I don't know if rdflib-jsonld supports that.

Reply:
Just confirmed via wireshark that it is indeed calling out each time. I unplugged my network connections and the next page load failed with:
```
URLError at /members/
<urlopen error [Errno 8] nodename nor servname provided, or not known>
```
They are serving a `cache-control` header:
```
HTTP/2 200 
access-control-allow-origin: *
content-type: text/html
cache-control: public, max-age=600
vary: Accept, Accept-Encoding
x-cloud-trace-context: 5e9e0e5152f49f64d37c1817b5072f51
content-length: 5069
date: Tue, 21 May 2019 15:24:42 GMT
server: Google Frontend
alt-svc: quic=":443"; ma=2592000; v="46,44,43,39"
```

I also noticed that `server` is `Google Frontend`; not sure what's up with that. In Wireshark my packet's destination IP resolved to a Google block. Google doesn't _own_ schema.org, do they?